sklearn/decomposition/tests/test_sparse_pca.py

```diff
@@ -70,6 +71,10 @@ def test_fit_transform():
     spca_lasso.fit(Y)
     assert_array_almost_equal(spca_lasso.components_, spca_lars.components_)
 
+    # Test calling tranform with deprecated ridge_alpha parameter
+    ridge_alpha = 0.01
+    assert_warns(DeprecationWarning, spca_lars.transform, Y, ridge_alpha)
```

Comment:
Also added an example where `ridge_alpha=None` to make sure it's handled properly